lte/gateway/docker/services/python/Dockerfile

```diff
@@ -76,6 +76,7 @@ COPY \
     $MAGMA_ROOT/lte/gateway/python/
 COPY ./lte/gateway/python/load_tests $MAGMA_ROOT/lte/gateway/python/load_tests
 COPY ./lte/gateway/python/scripts $MAGMA_ROOT/lte/gateway/python/scripts
+COPY ./lte/gateway/python/dhcp_helper_cli $MAGMA_ROOT/lte/gateway/python/dhcp_helper_cli
```

Comment:
This also copies tests etc - do we want this. But I guess this also happens in the other copy steps.